swift/Sources/FlatBuffers/ByteBuffer.swift

```diff
@@ -196,11 +205,13 @@ public struct ByteBuffer {
   init(
     memory: UnsafeMutableRawPointer,
     count: Int,
-    removing removeBytes: Int)
+    removing removeBytes: Int,
+    allowReadingUnalignedBuffers allowUnalignedBuffers: Bool = false)
```

Comment:
I guess in this case new storage is allocated with proper alignment and the unaligned access is not a case. So it seems there should be just `allowReadingUnalignedBuffers = false` in that `init` Or I miss something?

Reply:
Åh, true! I'll need to add a test for this actually since I just wrote it without consideration honestly. Because if copying an unaligned buffer to an aligned one might present issues when copying instead of reading (not sure how swift will handle it)

Reply:
Fixed